org.eclipse.wb.core/src/org/eclipse/wb/internal/core/editor/constants/IEditorPreferenceConstants.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2021, 2021 DSA Daten- und Systemtechnik GmbH. (https://www.dsa.de)
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Marcel du Preez   - initial implementation
+*********************************************************************************/
+package org.eclipse.wb.internal.core.editor.constants;

```

Comment:
As we expect clients to use thsi constant I would move it to the package package org.eclipse.wb.core.editor.constants;